ctapipe/coordinates/disp.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
This module really has nothing to do with disp per-se, so I'd not call it "disp.py".

Maybe these functions are better at `ctapipe.reco.preprocessing`?

Reply:
Done